website/content/docs/concepts/transparent-sessions.mdx

```diff
@@ -88,6 +88,7 @@ Refer to the following table for known issues that may affect the public beta:
 | Boundary Client Agent resumes on reboot | If the Client Agent is paused and the machine is rebooted, the Client Agent will be resumed after the reboot. |
 | Single-word aliases do not work on Windows | If you create an alias consisting of a single word without a dot (`.`), the alias will not work on Windows. |
 | Windows installer does not support partial installations | The Windows installer fails to start the Client Agent if the Desktop client is not installed at the same time. |
+| Alias connection failures inside containers/VMs | Transparent Sessions rely on network access to the local network of the computer it is running on. Network enclaves such as those created by Docker containers and VMs cannot reach this network. |
```

Comment:
```suggestion
| Alias connection failures inside containers/VMs | Transparent sessions rely on network access to the local network of the computer it is running on. Network enclaves such as those created by Docker containers and VMs cannot reach this network. |
```

Reply:
Is "it" in this sentence referring to Boundary or the session itself? I'd like to be more precise here, if we could:

> Transparent sessions rely on network access to the local network of the computer _it_ is running on.

Reply:
Rewrote this to:
> Using transparent sessions rely on network access to the local network of the computer the client agent is running on.